src/settings/nursery.rs

```diff
@@ -0,0 +1,17 @@
+use crate::registry::Rule;
+
+/// Rules for which a reason is returned here are disabled by default unless they are explicitly selected.
+/// So for example `one-blank-line-before-class` (D203) will not be selected by "D" or "ALL" since it's listed here.
+/// The returned reason is documented in the autogenerated README as well as reported by the explain command.
+pub fn nursery_reason(rule: &Rule) -> Option<&'static str> {
+    match rule {
+        Rule::OneBlankLineBeforeClass => Some("Conflicts with PEP 257."),
+        Rule::MultiLineSummaryFirstLine => Some("Not part of PEP 257 and conflicts with D213."),
+        Rule::MultiLineSummarySecondLine => Some("Not part of PEP 257 and conflicts with D212."),
+        Rule::MagicValueComparison => {
+            Some("Comparisons to strings are generally not considered to be an anti-pattern.")
+        }
+        Rule::AssertUsed => Some("`assert` is generally not considered to be an anti-pattern."),
```

Comment:
I'd like to remove this, `Rule::MagicValueComparison`, and `Rule::MultiLineSummaryFirstLine` from this list for now. We can always add them later. But I don't want to couple the debate here to the removal of specific rules, apart from those that motivated the issue. The goal here is really to solve the infinite iteration loop for new users.

Reply:
> D212 (MultiLineSummaryFirstLine) is [part of a valid convention], and shouldn't be part of any sort of automatic disable list IMO

It is not part of PEP 257, which I'd consider to be the only default convention. I don't think it makes sense to enable convention-specific rules by default. Case in point: pydocstyle also does not enable D212 by default.